hamper/plugins/questions.py

```diff
@@ -101,6 +102,7 @@ def command(self, bot, comm, groups):
             r = random.random()
 
             flavor = [
+                (0.1, list(open('obliques.txt'))),
```

Comment:
This won't work. The lists here are for two options and three options, (ie, "a or b" vs "a or b or c"). You'll have to be a bit more clever to put a list here.

Also, is 0.1 too common? I guess we can always tweak it.